docs/events.rst

```diff
@@ -0,0 +1,93 @@
+======
+Events
+======
+
+.. warning::
+    Kubernetes itself contains a terminology conflict:
+    There are *events* when watching over the objects/resources,
+    such as in ``kubectl get pod --watch``.
+    And there are *events* as a built-on object kind,
+    as shown in ``kubectl describe pod ...`` in the "Events" section.
+    In this documentation, they are distinguished as the watch-events
+    and the k8s-events. This section describes the k8s-events only.
+
+Handled objects
+===============
+
+.. todo:: body arg must be optional, meaning the currently handled object.
+
+Kopf provides few tools to report the arbitrary information
```

Comment:
"provides some tools to report arbitrary information for the handled objects as Kubernetes events"